.github/workflows/lint.yml

```diff
@@ -99,6 +99,6 @@ jobs:
         with:
           save-if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/next' }}
       - name: Rebuild kernel errors
-        run: BUILD_KERNEL_ERRORS=1 cargo check -p miden-lib
+        run: BUILD_GENERATED_FILES_IN_SRC=1 cargo check -p miden-lib
```

Comment:
I think this is a more general check now. I'd do the following:

- Rename the task to something like "generated_files".
- Rename the step into "Rebuild generated files".
- Run it for all crates - not just for `miden-lib`.

Reply:
ok! 👌🏼